source/process/release-process.md

```diff
@@ -21,197 +21,177 @@ Pre-work for the current release begins at the code complete date of the previou
 
 ### B. (T-minus 15 working days) Cut-off for submitting major features
 
-No pull requests for major features should be **submitted** to the current release after this date (except if release manager decides to add "release-exception" label to Jira ticket).
```

Comment:
Line 15 (can't comment there): A bit of a nit: Do we hold to the 10am cutoff time for deadline days? Not sure we prioritize the specific time in practice, so it can slip to the following day(s), contributing to delays in finding / fixing bugs and cutting RCs. Would be good to keep the deadline time meaningful, either by moving items to 10am the following day (if people are working more towards EOD in their time zone). Or maybe we should change the cutoff time for deadline days to 5pm, either Toronto or SF time, if that better matches how people tend to work, but then make sure it's very clearly communicated and held to?